lib/cloudantBaseService.ts

```diff
@@ -209,4 +215,17 @@ export default abstract class CloudantBaseService extends BaseService {
     }
     return super.createRequest(parameters);
   }
+
+  private static setAgent(options: UserOptions) {
```

Comment:
I think we could be more "honest" about the naming of this function (or we could just move it inline in the ctor now)
```suggestion
  private static setDefaultAgentsIfUnset(options: UserOptions) {
```

Reply:
5b7faca